### PR TITLE
define helper method for handling scroll issues in ff

### PR DESCRIPTION
**Pull request by cryptoB0T**

### CODE REVIEW

File: app/javascript/App/index.js [hunk 1]
```diff
@@ -79,3 +82,4 @@ document.addEventListener('DOMContentLoaded', () => {
 })
 
 window.addEventListener('resize', debounce(setScreenSize), 400)
+window.addEventListener('resize', debounce(scrollHelper), 400)
```
Comment (hanchang) [inline app/javascript/App/index.js]: Dumb question - why is the scrollHelper used both app-wide here, as well as specifically in the NewsList?

Comment (cryptoB0T) [reply app/javascript/App/index.js]: it's a race condition. the app-wide component will mount first. it checks for existence of the html before applying the fix. if the app-wide check happens before the NewsList mounts then the helper won't work. 

we could probably get away with invoking the helper in just the CoinDrawer as that is a child of the NewsList. but i don't think that optimization will buy us noticeable gains in performance. on a slower machine the mounting can happen slowly. it's probably safer way to make sure the fix works most of the time.

File: app/javascript/App/scrollHelper.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export default (element) => {
```
[the remark this answers is not in the document]
Comment (cryptoB0T) [reply app/javascript/App/scrollHelper.js]: 👍

File: app/javascript/App/scrollHelper.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  // const newsfeedElem = document.querySelector('#newsfeed')
+  const newsBodyElem = document.querySelector('.selected-news-content')
+  const coinDrawerElem = document.querySelector('.coin-watch-list')
+  const bodyTag = document.querySelector('body').offsetHeight
+  const topNav = document.querySelector('.topnav').offsetHeight
+  const coinSearchHeight = 60
```
[the remark this answers is not in the document]
Comment (cryptoB0T) [reply app/javascript/App/scrollHelper.js]: you got it!

**Pull request merged**